c/opentx.h

```diff
@@ -162,6 +162,44 @@ void hash_cache_append2(HashCache *cache, uint8_t *buf, size_t len) {
   blake2b_update(&cache->state, buf, len);
 }
 
+int hash_cell_script(HashCache *cache, size_t index, size_t source,
+                     size_t field, uint16_t cell_mask) {
+  uint8_t script[OPENTX_SCRIPT_SIZE];
+  uint64_t len = OPENTX_SCRIPT_SIZE;
+  int err =
+      ckb_checked_load_cell_by_field(script, &len, 0, index, source, field);
+  CHECK(err);
```

Comment:
This is still wrong. Consider the error code (item missing) while handling type script.